web/controllers/notes_controller.ex

```diff
@@ -21,12 +21,26 @@ defmodule Notox.NoteController do
   def create(conn, %{"note" => note_params}) do
     changeset = Note.changeset(%Note{}, note_params)
     case Repo.insert(changeset) do
-      {:ok, note} ->
+      {:ok, _note} ->
```

Comment:
I don't have strong preferences on this. I might leave it as `_`, because this is a pretty common pattern, so it requires less expressivity, but I also generally think that being explicit isn't a bad thing. Up to you.